src/App.js

```diff
@@ -16,6 +18,8 @@ export default function App() {
       <Routes>
         {/* Rotas públicas */}
         <Route path="/" element={<Login />} />
+        <Route path="/cadastro" element={<Cadastro />} />
```

Comment:
Pra ir até o cadastro, o cadastre-se não tá linkado!!

Reply:
Testa ai pra tu ver, desde o começo...
eu comentei isso antes

Reply:
Aparentemente já manda pra cadastro, se for a tela principal. Vou marcar como resolvido.

Reply:
Aparentemente foi corrigido, vou marcar como revolvido.